### ISSUE TITLE
Table: Activable state with focusable class doesn't have appropriate focus outline color

### ISSUE THREAD

**Issue by N1XUS** [SAP/fundamental-styles#2840]
**Describe the bug**

Currently, we have an activable modifier and focusable modifier, which bring its own styling to the table rows and cells.
In case of combining these two, when row/cell is active (clicked), we will see wrong outline color:
AR:
![image](https://user-images.githubusercontent.com/6586561/138424185-ab6ec642-b3fb-4424-bcf3-2dd5f053c413.png)
ER:
![image](https://user-images.githubusercontent.com/6586561/138424219-a939b0bb-9cf0-48c1-897a-2bea5df78461.png)

> Is this issue related to a specific component?
Table
> What version of the Fundamental Library Styles are you using?
latest
